bundle/manifests/datasciencecluster.opendatahub.io_datascienceclusters.yaml

```diff
@@ -722,6 +722,9 @@ spec:
                         maxLength: 63
                         pattern: ^([a-z0-9]([-a-z0-9]*[a-z0-9])?)?$
                         type: string
+                        x-kubernetes-validations:
```

Comment:
why it has this change?

Reply:
as a result of this change: https://github.com/opendatahub-io/opendatahub-operator/pull/1753/files#diff-defbd6c9b8652e52df235f6ef0d84cd884fb32d2431dd827c5880136cf0f84dcR77-R84
🤔

Reply:
but it should be fixed for a bug to main, not bring it from a release PR 
@robotmaxtron

Reply:
@CFSNM has this immutable on workbench namespace tested in ODH nightly?

Reply:
@zdtsw It was added as a suggestion from @lphiri in https://github.com/opendatahub-io/opendatahub-operator/pull/1704#issuecomment-2697290815

Reply:
it should be part of https://github.com/opendatahub-io/opendatahub-operator/pull/1753/files#diff-ee90d4b036c81ca68adab0a26eda76782fbaf15b091f15200d8d91e171c1ff94 but it was not included, right?

Reply:
so it is correct to be here right?

Reply:
if not objection from team to keep as-is then post the message of label first then do the publish release tomorrow. my thoughts it is

Reply:
update about the label is done, I will go ahead with the release